### DISCUSSION TITLE
Paste website: component discoverability

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#484] [Ideas] [answered]
Already asked in a thread in Slack, but now that I've seen others have issues, I think there is a discoverability issue for Paste components.

On navigating to https://paste.twilio.design/components/, as an engineer I'd go straight to components to find React components. The issue is that components are actually spread out between only some of the sections, and without drilling down into each section can you find the component you're looking for. From a design system perspective I understand the division between these types of components such as layout and form elements, but from an engineering standpoint the difference is:
```tsx
import { Grid } from '@twilio-paste/core';
```
and
```tsx
import { Input } from '@twilio-paste/core';
```

Search bar won't help if I don't know what I'm searching for in the first place.

**Possible solutions**:

**antd**: https://ant.design/components/overview/

antd separates their components https://ant.design/components/overview/ and their design system https://ant.design/docs/spec/introduce. Doing so allows them to keep their components sidebar to only deal with components.

![image](https://user-images.githubusercontent.com/6936736/83908940-c12b8880-a71c-11ea-9856-74bb18c7f050.png)

**Material UI**: https://material-ui.com/getting-started/installation/

Material UI subcategorizes their components in one dropdown

![image](https://user-images.githubusercontent.com/6936736/83909004-d7d1df80-a71c-11ea-80c0-364a0311e17f.png)

**Comment by vankhoawin** [comment]
Related note: Icons are a component but from the sidebar it is not readily apparent

![image](https://user-images.githubusercontent.com/6936736/83909225-4151ee00-a71d-11ea-9d91-33c256bf9ae7.png)

**Comment by corinne-nagel** [comment, marked as answer]
This is awesome feedback! We have plans to revamp the Paste site navigation and would love to test/get your feedback as we do the research. Would you be open to giving us feedback as we get to the designing/researching?

**Reply by vankhoawin** [reply]
put me on the list :)
